### PR TITLE
fix: wrong stt vendor will raise ws notification

### PR DESCRIPTION
**Pull request by xquanluu**

### CODE REVIEW

File: lib/tasks/gather.js [hunk 1]
```diff
@@ -162,8 +162,9 @@ class TaskGather extends Task {
         alert_type: AlertType.STT_NOT_PROVISIONED,
         vendor: this.vendor
       }).catch((err) => this.logger.info({err}, 'Error generating alert for no stt'));
-
-      throw new Error(`no speech-to-text service credentials for ${this.vendor} have been configured`);
```
Comment (davehorton) [inline lib/tasks/gather.js:166]: I was thinking we still wanted to throw the error, just that before doing so we should send the jambonz:error message to the listening websocket client.

File: lib/tasks/gather.js [hunk 2]
```diff
@@ -385,7 +386,8 @@ class TaskGather extends Task {
         break;
 
       default:
-        throw new Error(`Invalid vendor ${this.vendor}`);
```
Comment (davehorton) [inline lib/tasks/gather.js:388]: same comment as above

**Review by davehorton — CHANGES_REQUESTED** [review]

File: lib/tasks/gather.js (hunk 1 above)
Comment (xquanluu) [reply lib/tasks/gather.js:166]: fixed

File: lib/tasks/gather.js (hunk 2 above)
Comment (xquanluu) [reply lib/tasks/gather.js:388]: fixed

**Review by davehorton — APPROVED** [review]

**Comment by davehorton** [comment]
In the future @xquanluu please put the issue number prefaced by hash in the commit notice, as that will put a nice link in the issue to the PR

**Pull request merged**